contracts/integrations/test/framework/assertions/moveStake.ts

```diff
@@ -221,31 +220,4 @@ export function moveStakeNonexistentPoolAssertion(
         },
     );
 }
-
-/**
- * Returns a FunctionAssertion for `moveStake` which assumes the input amount exceeds the moveable
- * amount of stake. Checks that the transaction reverts.
- */
-export function moveStakeInvalidAmountAssertion(
-    deployment: DeploymentManager,
-): FunctionAssertion<[StakeInfo, StakeInfo, BigNumber], void, void> {
-    return new FunctionAssertion<[StakeInfo, StakeInfo, BigNumber], void, void>(
-        deployment.staking.stakingWrapper,
-        'moveStake',
-        {
-            after: async (_beforeInfo: void, result: FunctionResult) => {
-                // Ensure that the tx reverted.
-                expect(result.success).to.be.false();
-
-                // This isn't ideal but unfortunately there are several different revert errors that
-                // can be triggered by trying to move more stake than possible.
-                expect(
```

Comment:
Oh I meant just remove this `expect()`, not the whole assertion :innocent: 
Probably nbd either way though.